server/server.cpp

```diff
@@ -0,0 +1,136 @@
+#include "server.h"
+#include <QtWidgets>
+#include <QtNetwork>
+#include <QtCore>
+#include <QMessageBox>
+#include <QFile>
+#include <QTextStream>
+#include <QStringList>
+#include <QObject>
+#include <QDebug>
+
+
+
+Server::Server(QWidget *parent)
+    : QDialog(parent)
+    ,statusLabel(new QLabel)
+{
+    setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint );
+    statusLabel-> setTextInteractionFlags(Qt::TextBrowserInteraction);
+    initServer();
```

Comment:
assume we init the server, but file cannot be read or it has no data to share, does this server has anything to do?